cli/utils.go

```
@@ -97,3 +99,134 @@ func convertMetadata(m string) (map[string]interface{}, error) {
 	}
 	return nil, nil
 }
+
+func executeCommand(root *cobra.Command, v any, args ...string) (errorRes, out string, err error) {
+	buf := new(bytes.Buffer)
+	root.SetOut(buf)
+	root.SetErr(buf)
+	root.SetArgs(args)
+
+	o := os.Stdout
+	e := os.Stderr
+
+	r, w, _ := os.Pipe()
```

Don't ignore errors assert they are nil or return